### ISSUE TITLE
v3.0.0 Emojis Crash

### ISSUE THREAD

**Issue by searchy2** [bazookon/giphy_get#22]
Emojis no longer load on v3.0.0 due to this error: 

```dart
E/flutter (10673): [ERROR:flutter/lib/ui/ui_dart_state.cc(209)] Unhandled Exception: type 'Null' is not a subtype of type 'int' in type cast
E/flutter (10673): #0      new GiphyPagination.fromJson (package:giphy_get/src/client/models/collection.dart:54:43)
E/flutter (10673): #1      new GiphyCollection.fromJson (package:giphy_get/src/client/models/collection.dart:17:33)
E/flutter (10673): #2      GiphyClient._fetchCollection (package:giphy_get/src/client/client.dart:117:28)
E/flutter (10673): <asynchronous suspension>
E/flutter (10673): #3      _GiphyTabDetailState._loadMore (package:giphy_get/src/views/tab/giphy_tab_detail.dart:209:21)
E/flutter (10673): <asynchronous suspension>
E/flutter (10673): 
```

**Comment by Invectys** [comment]
+

**Comment by bazookon** [comment]
WIP

**Comment by bazookon** [comment]
Fixed v3.0.1
